extensions/azurecore/package.json

```diff
@@ -41,6 +41,39 @@
             "description": "%config.enablePublicCloudDescription%"
           }
         }
+      },
+      {
+        "type": "object",
+        "title": "Azure Account Configuration",
```

Comment:
You might want to have a localized title which is pulled from the `package.nls.json` for the new properties

Reply:
How would I reference that from here?

Reply:
Something similar to https://github.com/microsoft/azuredatastudio/pull/8984/files#diff-a7e46294cf1dea22425a9e834e76e256R25